app/screens/FindLocationType.js

```diff
@@ -65,11 +68,11 @@ const FindLocationType = ({ navigation, locations: { locationTypes = [] } }) =>
                 inputContainerStyle={s.filterInput}
                 containerStyle={{backgroundColor:theme.neutral,borderBottomWidth:0,borderTopWidth:0}}
             />
-            <FlatList
+            <FlatList {...keyboardDismissProp}
```

Comment:
I'm seeing onScrollBeginDrag and keyboardDissmissMode as props for Scrollview components. They're not listed as props for FlatList components. This is probably not doing anything. Mind if I remove @RyanTG ?

Reply:
Hmm. I think it's doing something. Without that, the keyboard does not get dismissed when you scroll. For example, go to "find by operator" and focus on the input so that the keyboard shows up, then scroll down the list. The keyboard doesn't get dismissed, but it does when the above prop is added.

Reply:
Ok, cool, I just read through more closely and see that FlatList inherits the props of Scrollview. Sorry about that. Good sanity check!